package.json

```diff
@@ -19,9 +19,10 @@
   "main": "./lib",
   "dependencies": {
     "buffer-writer": "1.0.1",
-    "packet-reader": "0.3.1",
     "js-string-escape": "1.0.1",
+    "packet-reader": "0.3.1",
     "pg-connection-string": "0.1.3",
+    "pg-native": "^2.2.0",
```

Comment:
Why this addition of pg-native ?

Reply:
Ah shit. The tests seem to be adding that when they are run. I'll remove it.